brave/brave-core
# Splittunnel brave vpn android

Resolves https://github.com/brave/brave-browser/issues/22938

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Purchase the vpn subscription
2. go to vpn settings menu
3. Open Split tunneling option
4. Select Chrome as excluded apps and click save.
5. turn on the vpn connection again
6. Go to ipinfo.io on brave application
7. it should show the information regarding connected vpn server
8. go to ipinfo.io in chrome application
9. It should show the ISP information

YOu can verify the smae with any other application you selected as excluded apps.
